_posts/status_update_2020_07_10.md

```diff
@@ -0,0 +1,53 @@
+layout: post
+title: "10th July 2020: IAC Team Status Update"
+author: adrianiurca
+categories:
+
+- team
+- status
+  tags:
+- releases
+- community_management
+- iis
+- tomcat
+- powershell
+- shoutouts
+
+---
+
+The IAC team has been working on a lot of things over the past week.
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+
+- [puppetlabs/iis](https://github.com/puppetlabs/puppetlabs-iis)(7.1.0)
+- [puppetlabs/java_ks](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
+- [puppetlabs/powershell](https://github.com/puppetlabs/puppetlabs-powershell)(4.0.0)
+
+## Community Shout outs for IAC
+
+We appreciate our community contributors as they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thanks to [Ben Ford](https://github.com/binford2k) for his great idea [feature](https://github.com/puppetlabs/community_management/pull/39) on community_management.
+
+Thanks to [Chris Denneen](https://github.com/cdenneen) for the contributions on pdk-templates.
+
+## Customer tickets this week
+
+The team works hard at Docker issue ([MODULES-10691](https://tickets.puppetlabs.com/browse/MODULES-10691)).
+We have a PR opened for this issue [PR](https://github.com/puppetlabs/puppetlabs-docker/pull/632).
+Many thanks to [Daiana](https://github.com/daianamezdrea) for managing this problem.
+
+## Michael and Sheena are getting back to DCS revamp work after a hard triage
+
+@TODO
```

Comment:
@michaeltlombardi Could you please confirm this . 
Will be working on a handcrafted automated end-to-end test which validates autogenerated modules are ready for production.